src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MaxNumber = 100;
```

Comment:
```suggestion
    private static final int MAX_NUMBER = 100;
```
constants should be named in upper case